cmd/server/main.go

```
@@ -0,0 +1,63 @@
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
+	"github.com/umekikazuya/logleaf/internal/infrastructure/dynamo"
+	"github.com/umekikazuya/logleaf/internal/interface/handler"
+	"github.com/umekikazuya/logleaf/internal/usecase"
+)
+
+func main() {
+	r := gin.Default()
+
+	godotenv.Load()
```

golangci-lint (1.64.8)

Error return value of `godotenv.Load` is not checked

(errcheck)